README.md

```diff
@@ -1,2 +1,36 @@
-# reactive-streams-scalajs
-Reactive Streams bindings for ScalaJS
+# Reactive Streams for Scala.js
+
+The purpose of Reactive Streams is to provide a standard for asynchronous stream processing with non-blocking backpressure.
+
+This repository manages the [Scala.js](http://www.scala-js.org/)
+builds of the API. For details and the TCK see
+[the JVM](https://github.com/reactive-streams/reactive-streams-jvm).
```

Comment:
Maybe. Would be cool. Somebody has to work on it though :-)

How would that TCK look like? Javascript doesn't have interfaces, so maybe it would make sense to define them with Typescript or something, having the same shape as these ones. And then we could talk about interop with Javascript libraries as well.

On the other hand the JS TCK needs to be built with a testing-library agnostic core, if we want to be able to integrate with it from Scala.js and ScalaTest / SBT. 

Maybe we could describe the *laws* as functions / methods returning [ES6 Promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise), which in Scala.js [can be converted](https://www.scala-js.org/doc/sjs-for-js/es6-to-scala-part3.html#futures) into `Future`. So if we don't rely on some specific testing framework to describe these laws, like Jasmine, QUnit, or whatever, could turn out to be very cool.

I would like these Scala.js interfaces published sooner, so I can depend on them. The issue is that in https://monix.io, the code being cross-compiled, I'm already testing for compatibility by using Java's TCK. Having the JS TCK would be great, but not really necessary.

So maybe we could release version `0.90` or maybe `1.0-M1` or something like that, having just these interfaces, so people [don't have to duplicate them](https://github.com/monixio/monix/tree/master/monix-execution/js/src/main/scala/org/reactivestreams) first and then maybe we can work on the JS TCK. I'd be happy to lend a hand, though I'm not sure about how much time I have.